apm-agent-core/src/main/java/co/elastic/apm/agent/logging/LoggingConfiguration.java

```diff
@@ -159,8 +159,53 @@ public void assertValid(Boolean value) {
         })
         .buildWithDefault(false);
 
+    private final ConfigurationOption<LogEcsReformatting> logEcsReformatting = ConfigurationOption.enumOption(LogEcsReformatting.class)
+        .key("log_ecs_reformatting")
+        .configurationCategory(LOGGING_CATEGORY)
+        .tags("added[1.22.0]", "experimental")
+        .description("Specifying whether and how the agent should automatically reformat application logs \n" +
+            "into ECS-compatible JSON, suitable for ingestion into Elasticsearch for further Log analysis. \n" +
```

Comment:
I have not tested the ref works but I think it makes sense to link to the ecs docs so people can learn more about it.
```suggestion
            "into {ecs-logging-ref}/index.html[ECS-compatible JSON], suitable for ingestion into Elasticsearch for further Log analysis. \n" +
```